formant_ros2_adapter/scripts/config.json

```diff
@@ -2,12 +2,12 @@
     "ros2_adapter_configuration": {
         "subscribers": [
             {
-                "ros2_topic": "/bird_output_compressed",
+                "ros2_topic": "/combined_imgs_compressed",
                 "formant_stream": "bird.image",
                 "ros2_message_type": "sensor_msgs/msg/CompressedImage"
             },
             {
-                "ros2_topic": "/windshield_camera/image_raw/compressed",
+                "ros2_topic": "/windshield_compressed_image",
```

Comment:
@Infinite-Echo Was this supposed to change? I couldnt find the topic windshield_compressed_image referenced anywhere in the image_stitcher branch

Reply:
Whoops. That was left in from when I was originally going to combine windshield with another feed. I'll revert that.